### PR TITLE
docs(journey-maps): add several new examples for journey maps client.…

### PR DESCRIPTION
**Pull request by dev-ale**
… delete full example

### CODE REVIEW

File: src/components-examples/journey-maps/angular/journey-maps-style-options/journey-maps-style-options-example.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          </sbb-select>
+        </sbb-form-field>
+      </div>
+    </div>
+
+    <div formGroupName="styleVersion">
+      <div>
+        <span class="sbb-label">Style</span>
+        <sbb-radio-group class="sbb-checkbox-group-horizontal" formControlName="versionNumber">
+          <sbb-radio-button value="v1">v1</sbb-radio-button>
```
Comment (mstankala) [inline src/components-examples/journey-maps/angular/journey-maps-style-options/journey-maps-style-options-example.html:39]: I would label the v1 radio button (legacy)
```suggestion
          <sbb-radio-button value="v1">v1 (legacy)</sbb-radio-button>
```

File: src/components-examples/journey-maps/angular/journey-maps-map-navigation/journey-maps-map-navigation-example.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    [apiKey]="apiKey"
+    #advancedMap
+    language="en"
+    (mapCenterChange)="mapCenterChange.next($event)"
+    (mapBoundsChange)="mapBoundingBoxChange.next($event)"
+    (zoomLevelsChange)="zoomLevels = $event"
+  ></sbb-journey-maps>
+</div>
+<div class="sbb-navigation">
+  <div class="row-1">
```
Comment (mstankala) [inline src/components-examples/journey-maps/angular/journey-maps-map-navigation/journey-maps-map-navigation-example.html:15]: On narrow screens, the bounding box and zoom info is cut off. 
Can you force line breaks here?

File: src/components-examples/journey-maps/angular/journey-maps-routes-zones/journey-maps-routes-zones-example.ts [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+
+declare global {
+  interface Window {
+    JM_API_KEY: string;
+  }
+}
+
+/**
+ * @title Journey Maps - SBB Map Routes & Zones
+ * @includeExtraFiles ../shared/config.ts,../shared/markers.ts,../shared/journey/be-sh.ts,../shared/journey/zh-be_wyleregg.ts,../shared/journey/zh-sh_waldfriedhof.ts,../shared/routes/biel-lyss.ts,../shared/routes/bn-ls.ts,../shared/transfer/bern-indoor.ts,../shared/transfer/geneve-indoor.ts,../shared/transfer/luzern4-j.ts,../shared/transfer/zurich-indoor.ts,../shared/zone/bern-burgdorf.ts,../shared/zone/bs-bl.ts
```
Comment (mstankala) [inline src/components-examples/journey-maps/angular/journey-maps-routes-zones/journey-maps-routes-zones-example.ts:38]: This annotation @includeExtraFiles with routes and zones files, you need only in the Stackblitz example “SBB Map Routes & Zones”.
My suggestion: inside the shared/config.ts file, keep only the imports for zone routes only.
The remaining Consts values, copy directly into the concrete example:
- style_ids
- rail_colors
- ch_bounds
- poi_categories

File: src/components-examples/journey-maps/angular/index.ts [hunk 4]
```diff
@@ -11,10 +11,28 @@ import { SbbTabsModule } from '@sbb-esta/angular/tabs';
...
-import { JourneyMapsFullExample } from './journey-maps-full/journey-maps-full-example';
+import { JourneyMapsMapNavigationExample } from './journey-maps-map-navigation/journey-maps-map-navigation-example';
+import { JourneyMapsPoisMarkersExample } from './journey-maps-pois-markers/journey-maps-pois-markers-example';
+import { JourneyMapsRoutesZonesExample } from './journey-maps-routes-zones/journey-maps-routes-zones-example';
+import { JourneyMapsStyleOptionsExample } from './journey-maps-style-options/journey-maps-style-options-example';
+import { JourneyMapsUiOptionsExample } from './journey-maps-ui-options/journey-maps-ui-options-example';
 
-export { JourneyMapsBasicExample, JourneyMapsFullExample };
-const EXAMPLES = [JourneyMapsBasicExample, JourneyMapsFullExample];
+export {
```
Comment (mstankala) [inline src/components-examples/journey-maps/angular/index.ts:20]: Can we "force" somehow the order of the examples @kyubisation ?
I would like to habe the “SBB UI Options” as second example, after maps-basic.
Then “SBB Map Navigation” and the others.

**Review by mstankala — CHANGES_REQUESTED** [review]
please check my suggestions

**Review by mstankala — APPROVED** [review]
LGTM

**Comment by mhaertwig** [comment]
Since it's not a functional change of journey-maps itself, please use `docs(journey-maps): ...` as a PR title. This affects versioning.

**Review by mhaertwig — APPROVED** [review]

**Pull request merged**